src/app.service.ts

```diff
@@ -1,8 +1,12 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable } from "@nestjs/common";
 
 @Injectable()
 export class AppService {
-  getHello(): string {
-    return 'Hello World!';
+  getQuiz(): string {
```

Comment:
The same is here. Move to the quiz folder. 
- src
  - quiz
  app.controller.ts
  app.module.ts
  app.service.ts
  
  This is your folder structure. Every time use this structure for new endpoints.

Reply:
+ refactor done